NEWS.md

```diff
@@ -5,6 +5,11 @@
 * `seq_linter()` additionally lints on `1:n()` (from dplyr) 
   and `1:.N` (from data.table) (#1396, @IndrajeetPatil).
 
+## New features
+
+* `commented_code_linter()` now lints commented argument code, containing a trailing comma, as well (#386, @AshesITR).
```

Comment:
Understand better now. I guess we could also include a test that this extension applies regardless of whether the comment occurs inside a function call/definition, but it also smells like encoding false-positive-ish behavior, WDYT?

Reply:
What do you mean? The parse tree is oblivious to where the comment occurs.

Reply:
Right -- but I believe the intent of #386 was to only cover such comments inside a function call/definition. e.g. now we'll get false positives like:

```
# check the following for further details:
#   short-link/helpful-doc-1,
#   short-link/helpful-doc-2.
```

(IINM line 2 will newly lint, line 3 linted before)

Reply:
OTOH if the entire function call was block-commented, we'd have a bunch of false negatives.

I think we'd be better off by tightening the logic to require the commented code to potentially cause side effects, i.e. contain a function call or an assignment.

Reply:
> I think we'd be better off by tightening the logic to require the commented code to potentially cause side effects, i.e. contain a function call or an assignment.

I think I wasn't sure if you planned to attempt that in this PR. Maybe leave it as a TODO otherwise?

Reply:
I'd leave this for a later PR, there are already issues so not sure we need to add a TODO to the code.

Reply:
SGTM; resolved